src/pretix/base/reldate.py

```diff
@@ -143,17 +155,19 @@ def from_string(cls, input: str):
                     time = datetime.time(hour=int(timeparts[0]), minute=int(timeparts[1]), second=int(timeparts[2]))
                 try:
                     data = RelativeDate(
-                        days_before=int(parts[1] or 0),
+                        days=int(parts[1] or 0),
                         base_date_name=parts[3],
                         time=time,
-                        minutes_before=None
+                        minutes=None,
+                        is_after=parts[4] == "after",
```

Comment:
For the sake of completeness as above.

```suggestion
                        is_after=len(parts) > 4 and parts[4] == "after",
```